docs/_includes/sidebar.html

```diff
@@ -23,7 +23,7 @@
     <a href="{{ site.url | append: site.baseurl | append: localized_base_url  }}/tutorial/getting-started">
       <li class="title">{{ site.t[page.lang].start }}</li>
     </a>
-    <a href="{{ site.url | append: site.baseurl | append: localized_base_url  }}/api-docs/slack_bolt/" target="_blank">
```

Comment:
Removed `append: localized_base_url` as there is no localized version of docstrings